matrix/main.py

```diff
@@ -39,6 +39,8 @@ def setup(matrix, args=None):
     parser.add_argument("-L", "--log-name", nargs="*")
     parser.add_argument("-f", "--log-filter", nargs="*")
     parser.add_argument("-s", "--skin", choices=("tui", "raw"), default="tui")
+    parser.add_argument("-x", "--xunit", default=None, metavar='FILENAME',
```

Comment:
Yes, I was thinking we'd want the report to be independent of the screen.  For instance, when running under Jenkins, the console log would contain the raw screen output, while the report.xml file would contain the results.